rhel7/Dockerfile.pgo-backrest-repo.rhel7

```diff
@@ -21,13 +21,15 @@ RUN yum -y install \
 RUN groupadd pgbackrest -g 2000 && useradd pgbackrest -u 2000 -g 2000
 ADD bin/pgo-backrest-repo /usr/local/bin
 RUN chmod +x /usr/local/bin/pgo-backrest-repo.sh /usr/local/bin/archive-push-s3.sh \
-    && mkdir -p /opt/cpm/bin \
-    && chown -R pgbackrest:pgbackrest /opt/cpm
+    && mkdir -p /opt/cpm/bin /etc/pgbackrest \
+    && chown -R pgbackrest:pgbackrest /opt/cpm \
+	&& chown -R pgbackrest /etc/pgbackrest
```

Comment:
Any reason the `&&` don't line up?

Reply:
The lines referenced should now align properly.